### DISCUSSION TITLE
Disable Auto Generation of New Devices and Entities?

### DISCUSSION THREAD

**Discussion by jeremiah-mitchell** [dwainscheeren/dwains-lovelace-dashboard#797] [Q&A]
DD was great for helping me create my initial dashboard views and populating the various rooms with devices. Now that my dashboards are established though, I'm having trouble keeping them clean of miscellaneous devices and, in particular, entities. Is there a way to default any new items to disabled until I've had a chance to add what I need to the dashboards? I'm running into issues where the underlying entities change (maybe more sensors are added, things are renamed, integrations change, etc) and I get a bunch of spurious entries added to my room views.

I've searched through the documentation, forum threads, and config settings with no luck. Is that an option currently? If not, any recommendations on how people usually handle that situation? I have a fairly diverse set of devices, so as integrations are updated, I frequently see new things pop up.

Thanks!

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
